### ISSUE TITLE
When attempting to start a netgame from map number MAP33 or greater, the crispy-doom.exe and crispy-doom-setup.exe remain hanging as processes but having no windows.

### ISSUE THREAD

**Issue by Zodomaniac** [fabiangreffrath/crispy-doom#154]
Netgames have been broken, not sure when exactly. 2342 is the default UDP port, I just never changed that when testing netgames.

**Comment by Zodomaniac** [comment]
Well...I´m puzzled, copied the Feb 22 build files into the folder and then Feb 223 back, it works now...

**Comment by Zodomaniac** [comment]
I got it! When attempting to start a netgame from map number MAP33 or greater (like in this WAD 
[crudream.zip.txt](https://github.com/fabiangreffrath/crispy-doom/files/806007/crudream.zip.txt)
), the crispy-doom.exe and crispy-doom-setup.exe remain hanging as processes but having no windows. That's why I cannot launch a netgame before shutting down those processes in the task manager.

**Comment by fabiangreffrath** [comment]
Maps > 32 are currently not supported in network games:

https://github.com/fabiangreffrath/crispy-doom/blob/master/src/d_mode.c#L37

**Comment by fabiangreffrath** [comment]
That said, the game should exit with a proper error message!

Could you please reproduce this issue (i.e. the hanging of the engine) in Choco and report it over there? Thanks!

**Comment by Zodomaniac** [comment]
Gonna make another 40+ map PWAD when I get home because Crucified Dream is made for limit-removing ports and Choco won't handle it at all.

**Comment by fabiangreffrath** [comment]
Cool, thank you very much already!

**Comment by Zodomaniac** [comment]
Yes, Choco hangs as well in this situation. Here's a modified GothicDM I used to test, pushing 4 more maps into it (just duplicates of the existing ones) and warping the netgame to level 33 via 'Add extra parameters'.
P.S. Github keeps preventing me from uploading this flie XD

**Comment by Zodomaniac** [comment]
Opened the issue for Choco https://github.com/chocolate-doom/chocolate-doom/issues/875

**Comment by fabiangreffrath** [comment]
Great, thank you very much!

**Comment by fabiangreffrath** [comment]
So, this has been taken over to the Choco bug tracker more than two years ago.
